Calibration/IsolatedParticles/plugins/IsoTrackCalibration.cc

```diff
@@ -502,31 +705,37 @@ void IsoTrackCalibration::beginJob() {
   tree->Branch("t_eHcal",       &t_eHcal,       "t_eHcal/D");
   tree->Branch("t_hmaxNearP",   &t_hmaxNearP,   "t_hmaxNearP/D");
   tree->Branch("t_selectTk",    &t_selectTk,    "t_selectTk/O");
+  tree->Branch("t_qltyFlag",    &t_qltyFlag,    "t_qltyFlag/O");
   tree->Branch("t_qltyMissFlag",&t_qltyMissFlag,"t_qltyMissFlag/O");
   tree->Branch("t_qltyPVFlag",  &t_qltyPVFlag,  "t_qltyPVFlag/O)");
 
   t_DetIds      = new std::vector<unsigned int>();
```

Comment:
looks like a memory leak? perhaps t_DetIds does not need to be a pointer? (same with t_trgbits)